isaric/parsers/rapid_observations.toml

```diff
@@ -0,0 +1,782 @@
+[adtl.defs."admissionDateHierarchy"]
+combinedType = "firstNonNull"
+        fields = [
+            { field = "hostdat", description = "Admission date at this facility" },
+            { field = "cestdat", description = "Onset date of first/earliest symptom" },
+            { field = "dsstdat", description = "Date of enrolment" }
+        ]
+
+[adtl.defs."followupDateHierarchy"]
+combinedType = "firstNonNull"
+fields = [
+{ field = "flw_survey_date", description = "About you and your COVID-19 illness (if you're completing this survey on behalf of a child or adult that you care for, all the questions relate to their health and wellbeing)" },
+{ field = "flw2_survey_date", description = "About you and your COVID-19 illness (if you're completing this survey on behalf of a child or adult that you care for, all the questions relate to their health and wellbeing)" },
+{ field = "daily_date", description = "Date of follow-up" },
+{ field = "dsstdtc", description = "Outcome date" }
+]
+
+[[observation]]
+name = "haemorrhage"
+date = { ref = "admissionDateHierarchy" }
+phase = "admission"
+is_present = { field = "bleed_ceoccur_v2", ref = "Y/N/NK" }
+if = { bleed_ceoccur_v2 = { "!=" = 3 } }
+occurrence_period = 
```

Comment:
keys in TOML can't be left blank, you can comment it out meanwhile

Reply:
I will comment out the 'occurrence period'. I don't have values for these .. were they to be calculated automatically, based on two dates? or a selection from options? - to be discussed.